README.md

```diff
@@ -38,16 +44,21 @@ Since decorators are still in proposal state, you need to add the following plug
 $ yarn add -D babel-plugin-transform-typescript-metadata @babel/plugin-proposal-decorators babel-plugin-parameter-decorator
```

Comment:
I figured out that `@babel/core` is also needed. Not sure whether it's installed by default or not when using `next-cli`. If not, we should add it here.